From b9d4405e69dc6c69112a5546aea1c8d709336bac Mon Sep 17 00:00:00 2001
From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Fri, 4 Aug 2023 21:41:08 -0300
Subject: [PATCH] views/week-header: Check for NULL child

I'm not sure if this is actually correct, but I'm too tired to
think the week header through, so let's just be slightly more
protective for now.
---
 src/gui/views/gcal-week-header.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/gui/views/gcal-week-header.c b/src/gui/views/gcal-week-header.c
index a652618b..b1509cc7 100644
--- a/src/gui/views/gcal-week-header.c
+++ b/src/gui/views/gcal-week-header.c
@@ -764,6 +764,9 @@ apply_overflow_at_weekday (GcalWeekHeader *self,
 
   child = gtk_grid_get_child_at (self->grid, weekday, 3);
 
+  if (!child)
+    return;
+
   split_event_widget_at_column (self, child, weekday);
   gtk_widget_set_visible (child, FALSE);
 }
-- 
2.40.0

